packages/common/src/content/_internal.ts

```diff
@@ -189,18 +191,35 @@ const getSolutionUrl = (
     // NOTE: as per the above spreadsheet,
     // solution types are now in the Template family
     // but we don't send them to the route for initiative templates
```

Comment:
I would update this comment to reflect the current reality, especially since we now are sending them to the templates route